### ISSUE TITLE
SetupWizard not run (or needs improvement) on new profile initialization

### ISSUE THREAD

**Issue by crass** [GrapheneOS/os_issue_tracker#507]
Firstly, I may be incorrect that SetupWizard is not run when a new profile is switched to. The first screen I see when switching to a new user profile is one with the Graphene logo with a `Start` button at the bottom. When pressing `Start` I'm taken to the home screen for the user. Perhaps the SetupWizard is being run and, unlike the Owner setup, only has that one screen.

Either way, in order to restore to a new user profile, the user should go through a setup wizard like the one run when setting up the Owner account. Not all of the screens when setting up the Owner profile may be applicable. At a minimum, it would be nice to have the restore screen and the add finger print sequence of screens.

**Comment by thestinger** [comment]
The SetupWizard does already run on secondary user profiles. It doesn't currently go through most of the activities, but it runs. You'll need to be more specific about what you want. Seedvault is already covered by an existing issue.

**Comment by crass** [comment]
Apparently you didn't read the issue closely. This is not #332, although there is some overlap. 

I list 2 **specific** examples and noted that there were probably more.
1. Restore screen
2. Finger print sequence of screens

As you admit the SetupWizard does not go through most of the activities. Well that is precisely what this issue is about! This issue is about having the SetupWizard go through more activities when run from a non-Owner profile. What is not clear about that?

This is an issue about **SetupWizard** not any other upstream apps that it might use (eg. Seedvault). Now this issue **may** depend on #332 and Seedvault, but that doesn't change the previous fact. Also let me pose it a different way. If there is not issue with SetupWizard and only upstream projects, then why doesn't the SetupWizard for a secondary user goes through the finger print setup as is done for the Owner profile?

**Comment by thestinger** [comment]
Setup Wizard is an upstream project, and it does run itself for secondary profiles.

**Comment by crass** [comment]
Thanks for the clarification. I knew that it was a derivative of the lineage SetupWizard, but thought it was [forked for this project](https://github.com/GrapheneOS/platform_packages_apps_SetupWizard). So there's no mixups. Do you have a link to the upstream project?
